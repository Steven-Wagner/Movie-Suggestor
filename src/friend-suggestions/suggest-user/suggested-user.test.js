import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import SuggestedUser from './suggested-user';

describe('<SuggestedUser', () => {
    it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(
        <Router>
            <SuggestedUser user={{bio:"test bio", username:"test user", match_score: "3", user_id: "2"}} currentUser="1"/>
        </Router>,
        div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
        .create(
            <Router>
                <SuggestedUser user={{bio:"test bio", username:"test user", match_score: "3", user_id: "2"}} currentUser="1"/>
            </Router>)
        .toJSON();
        expect(tree).toMatchSnapshot();  
    });
});