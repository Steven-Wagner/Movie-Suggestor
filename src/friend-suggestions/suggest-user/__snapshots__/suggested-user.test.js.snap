// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuggestedUser renders the UI as expected 1`] = `
<section
  className="remote-edge"
>
  <header
    className="suggested-friend-header"
  >
    <h2>
      test user
    </h2>
    <p>
      test bio
    </p>
    <p>
      Match Score: 3
    </p>
  </header>
  <button
    className="remote-button"
    onClick={[Function]}
  >
    Follow
  </button>
</section>
`;
